src/Controls.java

```diff
@@ -0,0 +1,47 @@
+/*
+Triglav Library Database has two main classes.
+
+>> For interacting with the program through IDE IO, this is the main class.
+>> For interacting with the program through JavaFX GUI, MainLaunchGUI is the main class.
+ */
+
+import inventory.FalseDatabaseStarter;
+import inventory.Inventory;
+import inventory.LibraryDatabase;
```

Comment:
folder structure does not correspond to the package structure. Follow IntelliJ recommendations to move packages around.